Recipes/Recipes/All Recipes/Model/Recipes.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Recipes.swift
+//  Recipes
+//
+//  Created by Vermut xxx on 04.03.2024.
+//
+
+import Foundation
+
+/// Рецепт
+struct Recipes: Equatable {
+    /// Аватар рецепта
+    var avatarRecipie: String
+    /// Название рецепта
+    var titleRecipies: String
+    /// Время изготовления блюда
+    var cookingTimeTitle: String
+    /// Калории блюда
+    var caloriesTitle: String
+    
+    var isFavorite: Bool = false
+    
+    static func exampleRecipe() -> [Recipes] {
+        [
+        Recipes(
+            avatarRecipie: "fish_1",
+            titleRecipies: "Simple Fish And Corn",
+            cookingTimeTitle: "60",
+            caloriesTitle: "274"
+        ),
+        Recipes(
+            avatarRecipie: "fish_2",
+            titleRecipies: "Baked Fish with Lemon Herb Sauce",
+            cookingTimeTitle: "90",
+            caloriesTitle: "616"
+        ),
+        Recipes(
+            avatarRecipie: "fish_3",
+            titleRecipies: "Lemon and Chilli Fish Burrito",
+            cookingTimeTitle: "90",
+            caloriesTitle: "226"
+        ),
+        Recipes(
+            avatarRecipie: "fish_4",
+            titleRecipies: "Fast Roast Fish & Show Peas Recipes",
+            cookingTimeTitle: "80",
+            caloriesTitle: "94"
+        ),
+        Recipes(
+            avatarRecipie: "fish_5",
+            titleRecipies: "Salmon with Cantaloupe and Fried Shallots",
+            cookingTimeTitle: "100",
+            caloriesTitle: "410"
+        ),
+        Recipes(
+            avatarRecipie: "fish_6",
+            titleRecipies: "Chilli and Tomato Fish",
+            cookingTimeTitle: "100",
+            caloriesTitle: "174"
+        )]
+    }
+    
+    static let allRecipes: [Recipes] = exampleRecipe()
+        
+    /// свойство типа с избранными рецептами
+        static var favoritesRecipes = [Recipes(
+            avatarRecipie: "fish_1",
+            titleRecipies: "Simple Fish And Corn",
+            cookingTimeTitle: "60",
+            caloriesTitle: "274"
+        ),
+        Recipes(
+            avatarRecipie: "fish_2",
+            titleRecipies: "Baked Fish with Lemon Herb Sauce",
+            cookingTimeTitle: "90",
+            caloriesTitle: "616"
+        )]
+
+//        /// метод типа для получения данных
+//        static func getRecipes(category: DishCategory) -> [RecipeDescription] {
```

Comment:
```suggestion
//        static func getRecipes(category: DishCategory) -> [RecipeDescription] {
```
закоменченые убираем